tests/foreman/ui/test_reporttemplates.py

```diff
@@ -20,6 +20,7 @@
 import json
 import os
 from pathlib import PurePath
+from pathlib import Path
```

Comment:
Why import Path when PurePath is already imported?

Reply:
Because it's a different class? I even tried to check if I am missing something and: `E       NameError: name 'Path' is not defined`